jenkins/ocp-config/deploy/jenkins-master.yml

```diff
@@ -198,6 +200,8 @@ objects:
               value: '${DOCKER_REGISTRY}'
             - name: BITBUCKET_URL
               value: '${BITBUCKET_URL}'
+            - name: CLUSTER_GROUPS
```

Comment:
@stitakis I'd call it `ODS_CLUSTER_GROUPS` to be consistent with line 65

Reply:
sure, np